lbz/dev/server.py

```diff
@@ -24,19 +26,21 @@ class MyLambdaDevHandler(BaseHTTPRequestHandler, metaclass=ABCMeta):
     @property
     @abstractmethod
     def cls(self) -> Type[Resource]:
-        pass
+        """
+        DevServer needs to inherit a Resource.
```

Comment:
This is a misleading sentence as the current pattern does not rely on inheriting: resource is just assigned to `cls` on the class declaration level.

In the past, I wanted to remove this `cls` abstract method at all, or at least write a proper TODO because things like this need to be done simpler (developer-friendly).